overlord/configstate/configcore/refresh.go

```diff
@@ -27,6 +27,17 @@ import (
 )
 
 func validateRefreshSchedule(tr Conf) error {
+	refreshTimerStr, err := coreCfg(tr, "refresh.timer")
+	if err != nil {
+		return err
+	}
+
```

Comment:
(nitpick^99) I would remove the \n here to make it clearer that this block belongs together (read the new-style config option, try to parse it).